src/transformers/modeling_flax_utils.py

```diff
@@ -170,46 +172,57 @@ class FlaxPreTrainedModel(PushToHubMixin, FlaxGenerationMixin):
         - **main_input_name** (`str`) -- The name of the principal input to the model (often `input_ids` for NLP
           models, `pixel_values` for vision models and `input_values` for speech models).
     """
-    config_class = None
-    base_model_prefix = ""
-    main_input_name = "input_ids"
+    # class attributes
+    # Ideally we could just use ClassVars here to indicate the types, but src/transformers/utils/import_utils.py
```

Comment:
Interesting! Hadn't used ClassVars for type checking before in transformers. Perhaps for this PR we can go for the simpler `else` statement and fix the ClassVars issue in a follow-up PR? E.g. collapse it all down to:
```python
     config_class = None
     base_model_prefix = None
     main_input_name = "input_ids"
```